contrib/IECoreUSD/src/IECoreUSD/USDScene.cpp

```diff
@@ -1020,6 +1019,65 @@ IECoreScene::CurvesPrimitivePtr convertPrimitive( pxr::UsdGeomCurves curves, pxr
 	return newCurves;
 }
 
+IECore::ConstObjectPtr convertCamera( pxr::UsdGeomCamera camera, pxr::UsdTimeCode time)
+{
+    float horizontalAper=20.955,  horizontalOff = 0.0,  verticalAper =15.2908, verticalOff = 0.0;
+    camera.GetHorizontalApertureAttr().Get(&horizontalAper, time);
+    camera.GetVerticalApertureAttr().Get(&verticalAper, time);
+    camera.GetHorizontalApertureOffsetAttr().Get(&horizontalOff, time);
+    camera.GetVerticalApertureOffsetAttr().Get(&verticalOff, time);
+
+    IECoreScene::CameraPtr result = new IECoreScene::Camera;
+    pxr::TfToken  projToken;
+    camera.GetProjectionAttr().Get(&projToken);
+
+    if(projToken == pxr::UsdGeomTokens->perspective)
+    {
+        result->setProjection("perspective");
+
+        // We store focalLength and aperture in arbitary units.  USD uses tenths
+        // of scene units
+        float scale = 10.0f * result->getFocalLengthWorldScale();
```

Comment:
I think because of the USD assumption, this can just be:

   `result->setFocalLengthWorldScale( 0.1f );`

I think this is working right now as the default for a newly created `IECoreScene::Camera` is `0.1` so `scale` will always end up just being `1.0f`. 

I noticed that USD allows you to [specify the linear encoding units](https://graphics.pixar.com/usd/docs/api/group___usd_geom_linear_units__group.html) in the file. I had a quick chat with @johnhaddon and we can leave supporting this for another time (as it needs some thought on the right way to do it), so just setting the 'assumed' scale is fine for now, unless it's going to cause issues for you.